### ISSUE TITLE
Confusing error message with duplicate instance creation requests

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#343]
Originally reported of Google Code with ID 287.

```
If two instance creation requests are submitted in close order, it can happen that one grabs the lock for the new name, while the other hopes it can grab the lock, and then fails later with:

      Result:
        OpPrereqError
        [Couldn't add locks (instance2.example.com), probably because of a race condition with another job, who added them first, internal_error]

This is not an internal error per se, just "optimistic lock failing to acquire", so the message should be reworded appropriately and the error code changed from internal_error to not_unique.
```
Originally added on 2012-10-30 20:49:10 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed in commit a9d40c65ed9a28da5e8ca62b670cc979a17a75d0, will be released in 2.6.2
```
Originally added on 2012-11-01 13:11:41 +0000 UTC.

Changed State: Fixed
Added to Milestone: Release2.6

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as released the bugs fixed in 2.6.2.
```
Originally added on 2013-01-11 12:13:05 +0000 UTC.

Changed State: Released
